### PR TITLE
Upgrade rocketmq 4.9.3 .

### LINKED ISSUE #2542: [RocketMQ]Update RocketMQ to 4.9.3 in Spring Cloud Alibaba 2.2.x branch
我们鼓励使用英文，如果不能直接使用，可以使用翻译软件，您仍旧可以保留中文原文。
We recommend using English. If you are non-native English speaker, you can use the translation software.

Which Component
eg. RocketMQ

Is your feature request related to a problem? Please describe.
We are prepared to release a new version of Spring Cloud Alibaba in 2.2.x branch, we are plan to update some components.

Describe the solution you'd like
Update RocketMQ to 4.9.3 in Spring Cloud Alibaba 2.2.x branch, and text the reliability.

### PR DESCRIPTION
**Pull request by misselvexu**
### Describe what this PR does / why we need it

Upgrade rocketmq 4.9.3 .

### Does this pull request fix one issue?

Fixes #2542 .

### Describe how you did it
- Upgrade RocketMQ Version in `spring-cloud-alibaba-dependencies/pom.xml` .
- Improve rocketmq stream Module pom's dependencies.

### Describe how to verify it
I used the project's(https://github.com/alibaba/spring-cloud-alibaba/tree/master/spring-cloud-alibaba-examples/rocketmq-example) examples to finish the compatibility issues after updating the version .

### Special notes for reviews
none .

### CODE REVIEW

File: spring-cloud-alibaba-starters/spring-cloud-starter-stream-rocketmq/src/main/java/com/alibaba/cloud/stream/binder/rocketmq/autoconfigurate/RocketMQBinderAutoConfiguration.java [hunk 1]
```diff
@@ -65,4 +59,16 @@ public RocketMQMessageChannelBinder rocketMQMessageChannelBinder(
 				extendedBindingProperties, provisioningProvider);
 	}
 
+	@Configuration(proxyBeanMethods = false)
+	@ConditionalOnClass(HealthIndicator.class)
+	@ConditionalOnEnabledHealthIndicator("rocketmq")
+	static class KafkaBinderHealthIndicatorConfiguration {
```
Comment (steverao) [inline spring-cloud-alibaba-starters/spring-cloud-starter-stream-rocketmq/src/main/java/com/alibaba/cloud/stream/binder/rocketmq/autoconfigurate/RocketMQBinderAutoConfiguration.java]: Need to modify the name from KafkaBinderHealthIndicatorConfiguration to RocketMQBinderHealthIndicatorConfiguration

**Comment by steverao** [comment]
Is there any other thing we can do except upgrade version in RocketMQ 4.9.3?

**Comment by misselvexu** [comment]
> Is there any other thing we can do except upgrade version in RocketMQ 4.9.3?

I feel that there are not enough examples of RocketMQ related features for production in `spring-cloud-alibaba` .

**Comment by steverao** [comment]
> > Is there any other thing we can do except upgrade version in RocketMQ 4.9.3?
> 
> I feel that there are not enough examples of RocketMQ related features for production in `spring-cloud-alibaba` .

We did it in 2021.x branch and merge 2.2.x later.

**Comment by Sorieee** [comment]
> > Is there any other thing we can do except upgrade version in RocketMQ 4.9.3?
> 
> I feel that there are not enough examples of RocketMQ related features for production in `spring-cloud-alibaba` .

I have written some in 2021.x, will merge in #2570, welcome to improve it together

**Pull request merged**